edu-services/gradebook-service/api/src/java/org/sakaiproject/service/gradebook/shared/GradebookService.java

```diff
@@ -262,6 +262,17 @@ public GradeDefinition getGradeDefinitionForStudentForItem(String gradebookUid,
 	public CommentDefinition getAssignmentScoreComment(String gradebookUid, Long assignmentId, String studentUid)
 			throws GradebookNotFoundException, AssessmentNotFoundException;
 
+	/**
+	 *
+	 * @param gradebookUid
+	 * @param assignmentId
+	 * @param studentUid
+	 * @return
+	 * @throws GradebookNotFoundException
+	 * @throws AssessmentNotFoundException
+	 */
+	public boolean getAssignmentExcuse(String gradebookUid, Long assignmentId, String studentUid)
```

Comment:
The intent of this method isn't clear (though return type makes it apparent). Are you getting the assignment excuse or if the assignment is excused? Method name could be improved here.